multicluster/controllers/multicluster/commonarea/resourceimport_controller.go

```diff
@@ -339,6 +348,131 @@ func (r *ResourceImportReconciler) handleResImpDeleteForEndpoints(ctx context.Co
 	return ctrl.Result{}, nil
 }
 
+func (r *ResourceImportReconciler) handleResImpUpdateForClusterNetworkPolicy(ctx context.Context, resImp *multiclusterv1alpha1.ResourceImport) (ctrl.Result, error) {
+	acnpImpName := types.NamespacedName{
+		Namespace: "",
+		Name:      resImp.Spec.Name,
+	}
+	acnpName := types.NamespacedName{
+		Namespace: "",
+		Name:      common.AntreaMCSPrefix + resImp.Spec.Name,
+	}
+	klog.InfoS("Updating ACNP and ACNPImport corresponding to ResourceImport",
+		"acnp", acnpName.String(), "resourceimport", klog.KObj(resImp))
+
+	acnp := &v1alpha1.ClusterNetworkPolicy{}
+	err := r.localClusterClient.Get(ctx, acnpName, acnp)
+	acnpNotFound := apierrors.IsNotFound(err)
+	if err != nil && !acnpNotFound {
+		return ctrl.Result{}, err
+	}
+	if !acnpNotFound {
+		if _, ok := acnp.Annotations[common.AntreaMCACNPAnnotation]; !ok {
+			err := errors.New("unable to import Antrea ClusterNetworkPolicy which conflicts with existing one")
+			klog.ErrorS(err, "", "acnp", klog.KObj(acnp))
+			return ctrl.Result{}, err
+		}
+	}
+	acnpObj := getMCAntreaClusterPolicy(resImp)
+	tierKind, tierName := &v1alpha1.Tier{}, acnpObj.Spec.Tier
+	err = r.localClusterClient.Get(ctx, types.NamespacedName{Namespace: "", Name: tierName}, tierKind)
+	tierNotFound := apierrors.IsNotFound(err)
+	if !tierNotFound {
+		if acnpNotFound {
+			if err = r.localClusterClient.Create(ctx, acnpObj, &client.CreateOptions{}); err != nil {
+				klog.ErrorS(err, "failed to create imported Antrea ClusterNetworkPolicy", "acnp", klog.KObj(acnpObj))
+				return ctrl.Result{}, err
+			}
+		} else if !apiequality.Semantic.DeepEqual(acnp.Spec, acnpObj.Spec) {
+			acnp.Spec = acnpObj.Spec
+			if err = r.localClusterClient.Update(ctx, acnp, &client.UpdateOptions{}); err != nil {
+				klog.ErrorS(err, "failed to update imported Antrea ClusterNetworkPolicy", "acnp", klog.KObj(acnpObj))
```

Comment:
please capitalize the first word of log message, this kind of issue is quite a lot in mc codes. you can focus on new added one, I will submit a standalone PR to fix them all later. thanks.

Reply:
Cleaned up all the places in controllers that I can find